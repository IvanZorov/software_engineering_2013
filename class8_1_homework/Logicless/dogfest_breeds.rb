require 'csv'

i =0
name = ""
all = []
names = []

while i!= (ARGV.length)
 if ARGV[i] != nil
  names << ARGV[i]
 end
 i+=1
end

for i in 0..names.length-1 do
 names[i] = names[i].downcase
 names[i][0] = names[i][0].upcase
 name += names[i] + " "
end
name[name.length-1] = ''

CSV.foreach("dogfest.csv") do |r|
 if r[2] == name
  all << r
 end
end

all = all.sort
CSV.open("dogfest_breeds_res.csv", "w" ) do |c|
 all.each do |e|
  c << e
 
 end
end
