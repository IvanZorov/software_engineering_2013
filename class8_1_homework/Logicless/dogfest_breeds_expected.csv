6,AUGUST,Pointer,2,Seff,Hashimi
7,AUGUSTUS,Pointer,6,Viktor,Zorov
8,AUREOLE,Pointer,11,Ivo,Hristov
